packages/flutter/lib/src/material/slider.dart

```diff
@@ -1932,12 +1943,14 @@ class _RenderValueIndicator extends RenderBox with RelayoutWhenSystemFontsChange
 }
 
 class _SliderDefaultsM2 extends SliderThemeData {
-  _SliderDefaultsM2(this.context)
-    : _colors = Theme.of(context).colorScheme,
+  _SliderDefaultsM2({ required this.context, required SliderThemeData sliderTheme })
```

Comment:
Can the SliderThemeData not be acquired like the ThemeData is below using the context?

Reply:
Great idea! Removing `sliderTheme` parameter.